_inc/lib/core-api/class.jetpack-core-api-module-endpoints.php

```diff
@@ -1041,11 +1043,76 @@ private function _process_onboarding( $data ) {
 			}
 		}
 
+		// Add/update business address widget
+		if ( isset( $data['businessAddress'] ) ) {
+			//grab the first sidebar and proceed if it is present
+			$first_sidebar = self::get_first_sidebar();
+
+			if( $first_sidebar ) {
+				$title = wp_unslash( $data[ 'businessAddress' ][ 'name' ] );
+				$address = wp_unslash(
```

Comment:
Order seems a little weird to me - `city, state, street, zip`. In the original JPO plugin it was `street, city, state, zip` - https://github.com/Automattic/jetpack-onboarding/blob/master/class.jetpack-onboarding-end-points.php#L708.